core/src/main/java/cz/xtf/core/config/TestCaseContext.java

```diff
@@ -0,0 +1,47 @@
+package cz.xtf.core.config;
+
+import static cz.xtf.core.config.OpenShiftConfig.OPENSHIFT_NAMESPACE;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TestCaseContext {
+
+    /**
+     * Used only in case when test cases are executed in different namespaces when:
+     * -Dxtf.openshift.namespace.per.testcase=true
+     *
+     * This allows to track currently running test case for correct namespace mapping. This is used to automatically find
+     * namespace for running test case when
+     * creating {@link cz.xtf.core.openshift.OpenShift} instances.
+     */
+    private static String TEST_CASE_NAME;
+
+    private static final Map<String, String> testcaseToNamespaceMap = new HashMap<String, String>() {
+        {
+            // put default namespace into map to guarantee that it will be deleted with other namespaces
+            put(XTFConfig.get(OPENSHIFT_NAMESPACE), XTFConfig.get(OPENSHIFT_NAMESPACE));
```

Comment:
I still do not understand whole solution, but I wonder is really 2x `XTFConfig.get(OPENSHIFT_NAMESPACE)` correct here?  Seems to me suspicious.

Reply:
I'll double check it's still needed. I've changed the logic in `ProjectCreator` when deleting namespaces so it should not be needed any longer. However the deafult namespace will be still created as it's required to import image for checking metadata when `@SkipFor` annotation is used.

Reply:
Ok, I've removed it as it's no longer needed.